samcli/lib/utils/sam_logging.py

```diff
@@ -21,13 +28,18 @@ def configure_logger(logger, formatter, level):
         -------
         None
         """
-        log_stream_handler = logging.StreamHandler()
+        handlers = logger.handlers
+        if handlers:
+            log_stream_handler = handlers[0]
+        else:
+            log_stream_handler = logging.StreamHandler()
+            logger.addHandler(log_stream_handler)
```

Comment:
Do we not need to `addHandler()` in the if part?

Reply:
If there is already a handler, it indicates this method is called before. We only need to change the existing handler instead of creating a new one.

Reply:
So then we don't need line 32 and 33 above? Can we update this to only do the StreamHanlder if handler is None?